src/components/ApplicationGroup/Details/TriggerView/BulkCITrigger.tsx

```diff
@@ -237,23 +245,25 @@ const BulkCITrigger = ({
                     (!material.isBranchError && !material.isRepoError && !material.isRegex) ||
                     material.value !== '--'
                 ) {
-                    branchNames += `${branchNames ? ',' : ''}${material.value}`
+                    branchNames += `${branchNames ? ',' : ''}${getParsedBranchValuesForPlugin(material.value)}`
                 }
             }
-            return !branchNames
```

Comment:
Is it intentional?

Reply:
Fixed in https://github.com/devtron-labs/dashboard/pull/2174/files